jina/docker/hubio.py

```diff
@@ -61,6 +61,12 @@ def new(self):
 
         cookiecutter(self.args.template, overwrite_if_exists=self.args.overwrite, output_dir=self.args.output_dir)
 
+    def app(self):
```

Comment:
why this new interface?

Reply:
I see now

Reply:
`jina hub new` is for using cookiecutter to spin up template for jina-hub
`jina hub app` is for using cookiecutter to spin up `My First Jina App` using templates for jina core